### PR TITLE
Patterns/update application and setting

### LINKED ISSUE #192: Update readme & YAML for each pattern
Currently, there's a little bit of a inconsistency between the README for each pattern (for example, this guy: https://github.com/IBM-Watson/design-library/blob/develop/patterns/base/a/README.md) and the content model.

The README lists something called "Available Classes"
The content model calls that same information "Application Methods"

We need to resolve this inconsistency.

What that means is taking the "Available Classes" content *out* of the README and putting it in that pattern's YAML file under a variable called "Application Methods." This will enable us to treat the README and the application methods as separate information that can be styled and displayed differently.

(Also, at some point @Snugug will add a "Settings" variable to each YAML file.)

### PR DESCRIPTION
**Pull request by Snugug**
Resolves #192

### CODE REVIEW

File: patterns/base/body/pattern.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
 name: Body
 description: The document's body which contains all of its content
 displayTemplate: pattern
+applicationMethods:
+  - code: body
+    description: Applied to the `<body>` tag
+settings:
+  - setting: sans serif font
+    description: Font faimly for site
```
Comment (bobbinrobyn) [inline patterns/base/body/pattern.yml]: Font family

**Comment by bobbinrobyn** [comment]
:+1:

**Comment by ryanbrownhill** [comment]
:+1:

**Comment by poofichu** [comment]
:+1: :seedling:

**Pull request merged**